src/styles/variables.css

```
@@ -21,7 +21,9 @@
   --light-grey-pink: rgb(244, 238, 242); /* Used for user profile gradient start color */
   --light-ash: rgb(231, 242, 246); /* Used for user profile gradient end color */
   --light-purple: rgb(200, 214, 247); /* Used for some box shadows */
+  --light-purple-1: 200, 214, 247;
   --grey-purple: rgb(118, 116, 139); /* Used for the wallet connect button in the user navigation */
+  --grey-purple-1: 118, 116, 139; /* for color-mod replacement */
```

I really don't like the `-1` suffixed naming scheme.

Why not:
```suggestion
  --grey-purple-color-mod: 118, 116, 139;
```

Or yet another approach:
```css
--light-purple-raw: 200, 214, 247;
```

Which you can then pass both to `rgb` and `color-mod`:
```css
--light-purple: rgb(var(--light-purple));
...
background-color: rgba(var(--light-purple-raw), 0.33);
```

---

I notice elsewhere in others' prs that color-mod was working just fine and have now confirmed that for myself in this branch. Don't know what I was doing wrong to make me think it'd been deprecated. I've reverted everything in this branch back to normal. 

Apologies to anyone I may have confused by suggesting color-mod was deprecated😬